openstack/content-repo/templates/job.yaml

```diff
@@ -1,4 +1,47 @@
 {{- range $repo, $config := .Values.repos }}
+{{- if $config.schedule }}
```

Comment:
There is a lot of code duplication between the then-branch and else-branch that can be easily reduced (the pod spec is the same in both branches).